CHANGELOG.md

```diff
@@ -16,6 +16,12 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 ### Fixed
 - `n_task_params` now evaluates to 1 if `task_idx == 0`
 
+### Expired Deprecations (0.6.x)
```

Comment:
@AVHopp, @Scienfitz: not sure what is the best way to handle it in the changelog. Right now, I've just repeated the entries for the actual deprecations and added some headline. Any better ideas?

Reply:
hmm that feels weird to me to mention the exact same entry, because those entries are of the form `x has been replaced by y` which doesnt really answer the question `what deprecation has expired`

Why not simply 
```
### Expired Deprecations
- x1
- x2
- x3
```

or even better (more work)
```
### Expired Deprecations
- x1 class
- x2 module
- x3 function
```

Reply:
Fixed. If ok, pls resolve and accept. We'll then keep the PR pending until the release